packages/component/src/Transcript/KeyboardHelp.tsx

```diff
@@ -111,6 +111,9 @@ const KeyboardHelp: FC<{}> = () => {
             // "focusable" attribute is only available in IE11 and "tabIndex={-1}" does not work.
             focusable={false}
             role="presentation"
+            style={{
```

Comment:
Could you cache the `style` props into a constant to reduce wasted render?

Reply:
Good idea.

Reply:
Fixed this. Still trying to figure out how to write a test for high-contrast mode.